boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -203,24 +204,10 @@ CONTROLLER_GEN = controller-gen
 OPENAPI_GEN = openapi-gen
 endif
 
-
 .PHONY: op-generate
+## CRDv1beta1 is now deprecated
```

Comment:
IMO we don't need this comment. But if you must leave it, I would say "is no longer supported". It was deprecated for quite a while :)